code/modules/spells/spell_types/shadow_walk.dm

```diff
@@ -67,8 +68,12 @@
 	var/light_amount = T.get_lumcount()
 	if(light_amount > 0.2) // jaunt ends
 		end_jaunt(TRUE)
-	else if (light_amount < 0.2 && (!QDELETED(jaunter))) //heal in the dark
-		jaunter.heal_overall_damage(1,1, 0, BODYPART_ORGANIC)
+
+/obj/effect/dummy/phased_mob/shadow/proc/shadow_heal()
+	var/turf/T = get_turf(src)
+	var/light_amount = T.get_lumcount()
+	if (light_amount < 0.2 && (!QDELETED(jaunter))) //heal in the dark
+		jaunter.heal_overall_damage(2,2, 0, BODYPART_ORGANIC)
```

Comment:
does this stack with normal nightmare darkness healing, or does normal nightmare darkness healing not work while you're jaunting (leaving just the healing from this effect)?

Reply:
I'm pretty sure it stacks. I tested it out and found the healing rate to be good. I believe this procs every second much like the normal darkness healing.

Reply:
You should use delta time here instead of assuming constant time period.

Reply:
I'm not sure what you mean. On Proccess?

Reply:
Yes, scale the amount healed with time passed in there. Ideally with some heal per second define.